internal/infra/proxy_test.go

```diff
@@ -82,24 +82,21 @@ func TestNewProxy_customCert(t *testing.T) {
 	_ = addFileToArchive(tw, "/Dockerfile", 0644, proxyTestDockerfile)
 	_ = tw.Close()
 
-	tmp := ProxyImageName
-	defer func() {
-		ProxyImageName = tmp
-	}()
-	ProxyImageName = "curl-test"
-	resp, err := cli.ImageBuild(ctx, &buildContext, types.ImageBuildOptions{Tags: []string{ProxyImageName}})
+	proxyImageName := "curl-test"
+	resp, err := cli.ImageBuild(ctx, &buildContext, types.ImageBuildOptions{Tags: []string{proxyImageName}})
```

Comment:
Did you run `gofmt` on this? It doesn't look quite right to me, but I see CI runs gofmt via golangci-lint so I could be misremembering my `go`...

Reply:
Yes my editor runs it on save and I'm sure the linter would catch it if it wasn't proper gofmt.